src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16175.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if IOS || MACCATALYST // More tab is available in MAC and iOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue16175 : _IssuesUITest
+{
+	public Issue16175(TestDevice testDevice) : base(testDevice)
+	{
+	}
+	public override string Issue => "OnNavigatedTo was not triggered when tabs in More section";
+
+	[Test]
+	[Category(UITestCategories.TabbedPage)]
+	public void TabbedPageTabEventsTriggeredInMoreSection()
```

Comment:
This test is failing on iOS. Seems like the app is crashing:
`The app was expected to be running still, investigate as possible crash`
Could you review it?

Reply:
@jsuarezruiz The `TabbedPageTabEventsTriggeredInMoreSection` test passed locally without any issues. However, the other TabbedPage related failures appear to be related to my fix. I’ve updated the code accordingly. Please let me know if you have any further concerns.

Reply:
Seems to keep failing: 
**TabbedPageTabEventsTriggeredInMoreSection**

```
The app was expected to be running still, investigate as possible crash
TearDown : The app was expected to be running still, investigate as possible crash
```
Could you review it?